operations/helm/charts/mimir-distributed/values.yaml

```
@@ -2921,6 +2921,8 @@ gateway:
       httpSnippet: ""
       # -- Allows to set a custom resolver
       resolver: null
+      # -- Configures whether or not NGINX bind IPv6
+      listenIPv6: true
```

loki's config is `enableIPv6`. should we use the same?
```suggestion
      # -- Configures whether or not NGINX bind IPv6
      enableIPv6: true
```